### DISCUSSION TITLE
Pincell Multiphysics Tutorial Not Converging

### DISCUSSION THREAD

**Discussion by lliw-will** [neams-th-coe/cardinal#956] [Q&A] [answered]
I am trying to run the SFR pincell multi physics tutorial, but the steady state criteria for the transient solver to stop does not appear to be met. When viewing the tutorial files in paraview after forcibly quitting, the temperatures appear to be reaching a steady state at around 200-300 seconds. How many time steps should the simulation take to converge?

**Comment by aprilnovak** [comment, marked as answer]
Hi @lliw-will, we actually just recently fixed this -- MOOSE currently looks at *every* AuxVariable to be lower than the specified `steady_state_tolerance`. Because one of our aux variables is the tally standard deviation, there's a lot of noise which prevents convergence (only in a numerical sense, since the physics fields we care most about - temperature, power mean values, etc. do converge as expected). 

We just changed that tutorial to instead just use a fixed number of Picard iterations (https://github.com/neams-th-coe/cardinal/pull/953). Hopefully in the future MOOSE will be more flexible and allow different convergence tolerance levels for each variable. But in the meantime, if you update Cardinal that tutorial will be fixed now.

**Reply by lliw-will** [reply]
Thanks @aprilnovak, it's working properly now.
